man/systemd.network.xml

```diff
@@ -2402,6 +2402,14 @@
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>FairQueuingControlledDelayPacketLimit=</varname></term>
+        <listitem>
+          <para>Specifies the hard limit on the real queue size. When this limit is reached, incoming packets are
+          dropped. If the value is lowered, packets are dropped so that the new limit is met. Defaults to 10240 packets.</para>
```

Comment:
`10240`? I cannot find this value from the code...

Reply:
https://elixir.bootlin.com/linux/latest/source/net/sched/sch_fq_codel.c#L457